crates/swc/src/plugin.rs

```diff
@@ -86,6 +87,13 @@ impl RustPlugins {
         // transform.
         if let Some(plugins) = &self.plugins {
             for p in plugins {
+                let span = tracing::span!(
```

Comment:
Crateing manual span to track time to `serialize` json vs. actual plugin transform execution.